mnefun/_mnefun.py

```diff
@@ -2561,3 +2563,92 @@ def plot_chpi_snr_raw(raw, win_length, n_harmonics=None):
     plt.show()
 
     return fig
+
+
+def _rotation_angles(rot):
+    """return linear transformation for coordinate vector
+    From Matlab code written by Samu Taulu
+
+    Parameters
+    ----------
+    rot : array
+
+    """
+    beta = -np.arcsin(rot[2, 0])
+    cosb = np.cos(beta)
+    alpha = np.arcsin(rot[2,1]/cosb)
+    gamma = np.arccos(rot[0,0]/cosb)
+    return alpha, beta, gamma
+
+
+def plot_head_movement(pos):
+    """Visualize head movement and rotations from cHPI positions data"""
+    translations, rotations, times = head_pos_to_trans_rot_t(pos)
+
+    # find euclidean distance between successive translations
+    euc_dist = np.zeros(len(times) - 1)
+    for ii in np.arange(len(times) - 1):
+        euc_dist[ii] = euclidean(translations[ii], translations[ii + 1])
+
+    angles = np.zeros((len(times), 3))
+    for k in np.arange(0, len(times)):
+        angles[k] = _rotation_angles(rotations[k])
+
+    # plotting parameters
+    title_fontsize = 10
+    tick_fontsize = 10
+    label_fontsize = 10
+
+    # histogram of head displacement data
+    euc_dist *= 100
+    mu = euc_dist.mean()
+    sigma = euc_dist.std()
+    fig = plt.figure()
+    n, bins, patches = plt.hist(euc_dist, 100, normed=1,
+                                facecolor='blue',
+                                alpha=0.5)
+    y = mlab.normpdf(bins, mu, sigma)
+    l = plt.plot(bins, y, 'g--', linewidth=2)
+    plt.xlabel('Displacement (cm)')
+    plt.ylabel('Probability')
+    plt.title(
+        r'$\mathrm{Head\ displacement:}\ \mu=%.2f cm,\ \sigma=%.2f$' % (
+        mu, sigma))
+    #plt.axis([min(bins), max(bins), min(n), max(n)])
+    plt.grid(True)
+    plt.show()
+
+    # plot head movement
+    fig, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(3, 2, sharex=True,
+                                                             figsize=(20, 10))
+    ax1.plot(times, translations[:, 0] * 100)
```

Comment:
use a rcParams context manager to set the title, tick, etc. font sizes

http://matplotlib.org/api/matplotlib_configuration_api.html#matplotlib.rc_context